packages/map/src/helpers/generateColorsArray.test.ts

```diff
@@ -0,0 +1,18 @@
+import { generateColorsArray } from '../helpers/generateColorsArray'
```

Comment:
could you put this in a "tests" subfolder

Reply:
This is the actual function its pulling into the test. Are you suggesting that these functions should be in the test folder as well?

Reply:
yeah create a tests folder in helpers folder and put this generateColorsArray.test.ts file in there. putting the test folder in the folder its testing just reduces clutter for navigating the file tree.